api/v1alpha2/virtualmachine_conversion.go

```diff
@@ -58,6 +58,34 @@ func restore_v1alpha3_VirtualMachineImage(dst, src *vmopv1.VirtualMachine) {
 	dst.Spec.ImageName = src.Spec.ImageName
 }
 
+func restore_v1alpha3_VirtualMachineBootstrapLinuxPrep(dst, src *vmopv1.VirtualMachine) {
```

Comment:
@bryanv Unlike the previous change, I think we should just bite the bullet and update the v1a2 type to be nillable as well. It is the same format on-the-wire, and it can only result in compile-time issues for folks using the types (unlike before where an NPE was possible). This is because it's a `bool` and not a `struct`, meaning what access method that used to work would not now fail at compile time?

Reply:
Sgtm. I'll do that to avoid the potential for lossy data.

Reply:
Okay, the PR is updatd.